src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -2,20 +2,20 @@
   <ItemGroup>
     <RuntimeGroup Include="unix">
       <Parent>any</Parent>
-      <Architectures>x64;x86;arm;armel;arm64;mips64</Architectures>
+      <Architectures>x64;x86;arm;armel;arm64;mips64;s390x</Architectures>
     </RuntimeGroup>
     <RuntimeGroup Include="linux">
       <Parent>unix</Parent>
-      <Architectures>x64;x86;arm;armel;arm64;mips64</Architectures>
+      <Architectures>x64;x86;arm;armel;arm64;mips64;s390x</Architectures>
     </RuntimeGroup>
     <RuntimeGroup Include="linux-musl">
       <Parent>linux</Parent>
-      <Architectures>x64;x86;arm;armel;arm64</Architectures>
+      <Architectures>x64;x86;arm;armel;arm64;s390x</Architectures>
     </RuntimeGroup>
 
     <RuntimeGroup Include="alpine">
       <Parent>linux-musl</Parent>
-      <Architectures>x64;arm;arm64</Architectures>
+      <Architectures>x64;arm;arm64;s390x</Architectures>
```

Comment:
Is there really alpine for s390x ?

Reply:
Yes, we've been supporting alpine for a few years now, see e.g.
https://wiki.alpinelinux.org/wiki/S390x
https://alpinelinux.org/downloads/

Reply:
@uweigand is there a need for anything other than "rhel"?

Reply:
I wouldn't mind getting s390x support on Fedora too. I think it's better to keep it as general as possible.